### PR TITLE
feat(cost-map-widget): add api binding and fix styles

### PR DESCRIPTION
**Pull request by song2kim**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [X] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [X] `Error / Warning / Lint / Type`

### Description
* add api binding 
* fix styles

### CODE REVIEW

File: src/services/dashboards/widgets/cost-map/CostMapWidget.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
     <widget-frame v-bind="widgetFrameProps"
                   :error-mode="false"
+                  :widget-link="state.widgetLink"
```
Comment (yuda110) [inline src/services/dashboards/widgets/cost-map/CostMapWidget.vue:4]: quibble: There's no `widgetLink` yet...!🫢 No errors?

File: src/services/dashboards/widgets/cost-map/CostMapWidget.vue [hunk 2]
```diff
@@ -69,40 +67,71 @@ interface CostMapData {
...
 /* Api */
-const fetchData = async () => new Promise((resolve) => {
-    setTimeout(() => {
-        resolve(SAMPLE_RAW_DATA);
-    }, 1000);
-});
+const fetchData = async (): Promise<TreemapChartData[]> => {
+    try {
+        const apiQueryHelper = new ApiQueryHelper();
+        if (state.groupBy === 'project_id') apiQueryHelper.setFilters([{ k: 'project_id', v: null, o: '!=' }]);
```
Comment (yuda110) [inline src/services/dashboards/widgets/cost-map/CostMapWidget.vue:99]: todo: `groupBy` of this widget may be dynamical!

```suggestion
        if (state.groupBy === state.groupBy) apiQueryHelper.setFilters([{ k: state.groupBy, v: null, o: '!=' }]);
```

**Review by yuda110 — APPROVED** [review]
LGTM!! I left some comments!

**Review by sulmoJ — APPROVED** [review]
LGTM! I only checked conventions.

File: src/services/dashboards/widgets/cost-map/CostMapWidget.vue (hunk 1 above)
Comment (song2kim) [reply src/services/dashboards/widgets/cost-map/CostMapWidget.vue:4]: I need to comment  Thnx

**Pull request merged**